Library/Homebrew/cli/args.rb

```diff
@@ -1,11 +1,17 @@
-# typed: true # rubocop:disable Sorbet/StrictSigil
+# typed: strict
 # frozen_string_literal: true
 
 require "ostruct"
 
 module Homebrew
   module CLI
     class Args < OpenStruct
+      # FIXME: Enable cop again when https://github.com/sorbet/sorbet/issues/3532 is fixed.
+      # rubocop:disable Style/MutableConstant
+      OptionsType = T.type_alias { T::Array[[String, T.nilable(String), T.nilable(String), String, T::Boolean]] }
```

Comment:
Could use a comment describing what this is, since I don't know how to parse the block inside `T.type_alias`.

Reply:
Good suggestion! Here's what I have so far:
```ruby
      # Represents a processed option. The elements are:
      # 0: short option name (e.g. "-d")
      # 1: long option name (e.g. "--debug")
      # 2: 
      # 3: option description (e.g. "Print debugging information")
      # 4: whether the option is hidden)```
```
I can't figure out the element at index 2, it was `nil` for every combination of command and options i threw at it.

Reply:
Interesting. Does Sorbet throw a (static or runtime) error if we replace `2` with `NilClass`?

Also, I wonder if we could/should use a `T::Struct` here instead.

Reply:
Looks like it's never used anyway so we could remove it:

https://github.com/Homebrew/brew/blob/2a5eb025f1d36e41667bb22a564ce03cb1eceb48/Library/Homebrew/commands.rb#L202

https://github.com/Homebrew/brew/blob/2a5eb025f1d36e41667bb22a564ce03cb1eceb48/Library/Homebrew/manpages.rb#L101

Reply:
It's been yanked, PTAL